.travis.yml

```diff
@@ -16,13 +16,12 @@ os:
 
 node_js:
   - "10"
-  - "10.15"
-  - "10.19"
```

Comment:
I think this library is used in both cli and actions, so we need to keep `10.15` and `10.19`.